npm/win32-ia32-msvc/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "scylladb-javascript-driver-win32-ia32-msvc",
+  "version": "0.0.0",
+  "os": [
+    "win32"
+  ],
+  "cpu": [
+    "ia32"
+  ],
+  "main": "scylladb-javascript-driver.win32-ia32-msvc.node",
+  "files": [
+    "scylladb-javascript-driver.win32-ia32-msvc.node"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">= 10"
+  }
+}
```

Comment:
I don't know anything about these `package.json` files, hence the question: should they be versioned in the repo, or are they auto-generated on-demand?

Reply:
Bump

Reply:
It should be versioned, as this is JS version of `Cargo.toml`.
`package-lock.json` is JS version of `Cargo.lock` and it also should be versioned according to `npm help package-lock.json`